spec/stored_queries/taxonomy/taxonomy_search_species_strain_no_sort.yaml

```diff
@@ -53,6 +53,7 @@ query: |
   )
   FOR doc IN FULLTEXT(@@taxon_coll, @sciname_field, search_text__fulltext)
       FILTER @ts ? doc.created <= @ts AND doc.expired >= @ts : true
-      FILTER doc.rank IN ["species", "strain"] OR doc.strain
+      // note that doc.strain is deprecated TODO add link
```

Comment:
TODO before merge: add link to file in https://github.com/kbase/relation_engine_importers/pull/52 when that's merged

Reply:
done